whitepapers/0013_ccq.md

```diff
@@ -317,7 +317,42 @@ Currently the only supported query type on Solana is `sol_account`.
 
    - The `data_slice_offset` and `data_slice_length` are optional and specify the portion of the account data that should be returned.
 
-   - The `account_list` specifies a list of accounts to be batched into a single call. Each account in the list is a Solana `PublicKey`
+   - The `account_list` specifies a list of accounts to be batched into a single query. Each account in the list is a Solana `PublicKey`
+
+2. sol_pda (query type 5) - this query is used to ready data for one or more accounts on Solana based on their Program Derived Addresses.
+
+   ```go
+   u32         commitment_len
+   []byte      commitment
+   u64         min_context_slot
+   u64         data_slice_offset
+   u64         data_slice_length
+   u8          num_pdas
+   []byte      pda_list
+   ```
+
+   - The `commitment` is required and currently must be `finalized`.
+
+   - The `min_context_slot` is optional and specifies the minimum slot at which the request may be evaluated.
+
+   - The `data_slice_offset` and `data_slice_length` are optional and specify the portion of the account data that should be returned.
+
+   - The `pda_list` specifies a list of program derived addresses batched into a single query.
+
+     Each pda in the list is defined as follows:
+
+     ```go
+     [32]byte      program_address
+     u8            num_seeds
+     []byte        seed_data
+     ```
+
+     Each entry in `seed_data` is defined as follows:
+
+     ```go
+     u32           seed_len
```

Comment:
I wonder if Solana has a max length for seeds 🤔

Reply:
ah, `32` so looks like it can be a `u8` https://docs.rs/solana-program/latest/solana_program/pubkey/constant.MAX_SEED_LEN.html